packages/client/src/app/components/comments/comments.component.ts

```diff
@@ -17,7 +17,6 @@ import { CommentFormComponent } from './comment-form/comment-form.component';
 export class CommentsComponent implements OnInit {
   @Input() itemId: string; // The ID of the blog/work/newspost
   @Input() itemKind: string; // The kind of comments thread it is, between blogs/works/newsposts
-  @Input() itemUrl: string; // The item's URL
   @Input() pageNum: number; // The requested page number
   @Input() banlist?: any; // The banlist of the thread
```

Comment:
Can `banlist` have a more specific type? Or would we be fighting Angular too much?

Reply:
the banlist currently doesn't have a type because that input field is just a placeholder, the interface for the banlist still hasn't been made and wouldn't be useful right now because you can't ban people from comment threads yet. that's all getting done for 0.4.0